clang/include/clang/Driver/Options.td

```diff
@@ -3031,6 +3032,11 @@ defm skip_odr_check_in_gmf : BoolOption<"f", "skip-odr-check-in-gmf",
           "Perform ODR checks for decls in the global module fragment.">>,
   Group<f_Group>;
 
+def gen_reduced_bmi : Flag<["-"], "fgen-reduced-bmi">,
```

Comment:
If this is going to be a user-facing flag (even for a few releases) then I think it should be spelled in a way that make it modules-relatated (e.g. -fmodules-reduce-bmi, or -fmodules-minimise-bmi),  as you know I'm not a fan of increasing the already huge number of modules-related user-facing options... (but I'll comment on that separately)

Reply:
Done by using `-fmodules-reduce-bmi`